content/blog/2021/04/2021-04-12-jenkins-operator-sub-project.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Jenkins Operator becoming sub-project!"
+tags:
+- kubernetes
+- operator
+- contributing
+authors:
+- antoniaklja
+- sylwiabrant
+sig: docs
+opengraph:
+  image: /images/post-images/2021-04-jenkins-operator/tshirt-logo.jpg
+---
+
+We are happy to announce that Jenkins Operator officially became Jenkins sub-project.
+
+== What does it mean for the project?
+
+Becoming an official part of the Jenkins project was a major step towards better alignment with the overall Jenkins’ roadmap and more opportunities to increase adoption of the Jenkins Operator project.
+
+Finally, with a dedicated team of people actively contributing to the project we can engage with the wider community and participate in some of the Cloud-Native SIG meetings.
+*This opens a room for everyone to voice their opinions or start supporting the project.*
+
+We truly believe that this partnership will result in significant improvements to Jenkins Operator, as well as Jenkins ecosystem itself.
```

Comment:
Would it make sense to reference the open governance discussion somewhere?
It would be great to continue that though it is quite difficult without Red Hat at the moment.